### ISSUE TITLE
Final report

### ISSUE THREAD

**Issue by mbellitti** [mbellitti/wikiart-classifier#17]
Since the project is due on 2018-12-19, we should start to write something down. The simplest way is to use the README.md file we already have, and just point Pankaj to this repo. 

Let's put in some pretty pictures and once we're done we can send him the link.

**Comment by mohitpandey92** [comment]
[Here](https://github.com/mbellitti/wikiart-classifier/tree/report) is the latest version of ReadMe.

Somehow I can't access @mbellitti's SCC folder. I wanted to get the data about accuracy as a function of epochs to include in the ReadMe. 

Can @janpfsr  and @mbellitti look into it and make final changes to it?

**Comment by mbellitti** [comment]
I'm working on my QFT takehome final, so I can't work that much on this. I wrote a few things in the readme on the master branch, I didn't notice you made a report branch. Can you integrate those things?

I'll make the plot with the accuracy during training tomorrow.

**Comment by albrechtjan** [comment]
I have the same problem... I hope I finish today, then I can would try to make some more clustering attempts and create pictures for the report.

**Comment by mohitpandey92** [comment]
I understand. All the best for your QFT final!
I have merged "report" branch with "master". 
**To do**
1. Merge "Task Consideration" section with "What we did?" section
2.  XXX needs to be replaced with actual numbers.

**Comment by mbellitti** [comment]
There was something weird with the merge, should be fixed now. I'll work a bit on the report now.

**Comment by mohitpandey92** [comment]
In ReadMe, ""This sounds ambitious,...unsurprisingly we do not get accuracy larger than 50% even on the training set". @mbellitti, 50 % would be the natural scale to compare only if we had 2 labels with equal number of images. So, I suggest removing the comparison with 50 %. 
Not sure what is the natural scale to compare our accuracy.

**Comment by mbellitti** [comment]
It's true that there's not a clear comparison scale, so let's remove it from that section. But I would like to replace it with a sentence that gives an idea how how ill-defined the task is, so we don't expect 100% like in MNIST.

**Comment by mohitpandey92** [comment]
@janpfsr, the PCA and tSNE images in the "visualization" branch got removed. And as a result, ReadMe file doesn't show those images. Can you put the images again? Thanks!

**Comment by albrechtjan** [comment]
I put all the images in a designated folder. I'll relink it.

**Comment by mohitpandey92** [comment]
Never mind, I fixed it.
